include/mbedtls/error.h

```diff
@@ -80,6 +80,7 @@
  * CHACHA20  3                  0x0051-0x0055
  * POLY1305  3                  0x0057-0x005B
  * CHACHAPOLY 2 0x0054-0x0056
+ * ECDH      1                  0x001B-0x001D
```

Comment:
ECDH is above ECP and ECP is a high-level module, so ECDH should be a high-level module.

I'd prefer to refrain from adding a new module and declare all the errors in ECP.